### DISCUSSION TITLE
Build app for IOS simulator

### DISCUSSION THREAD

**Discussion by ihorkozar** [codemagic-ci-cd/codemagic-docs#2823] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

flutter build ios --debug --simulator

### Expected results

build what I can use for automation testing (Appium)

### Actual results

> flutter build ios --debug --simulator
Building ***** for simulator (ios)...
Running pod install...                                             50.6s
Running Xcode build...                                          
Xcode build done.                                           63.6s
Failed to build iOS app
Error (Xcode): Framework 'Pods_Runner' not found

Error (Xcode): Linker command failed with exit code 1 (use -v to see invocation)

Encountered error while building for simulator.

Build failed :|
Failed to build for iOS

### Build id (optional)

67337c2d29eee95a53cf3ad2

**Comment by dam-ease** [comment]
Hello @ihorkozar,
I tried this on my end and it works fine both locally and using Codemagic. Please can you try running this locally on your machine as well?

**Comment by ihorkozar** [comment]
Hi @dam-ease , on my local machine, I use flutter build ios --debug --simulator, and it works correctly. However, on Codemagic, when I add --simulator to the build section, I see the error mentioned in the initial message. If I remove --simulator, everything works fine.

**Reply by dam-ease** [reply]
I did a quick google search of the error message from your build and found a solution in [this SO thread](https://stackoverflow.com/questions/77304874/framework-pods-runner-not-found). Can you please try this?

**Reply by dam-ease** [reply]
Hello @ihorkozar,
Any update for me on this?If there is anything else I can assist you with, please don't hesitate to let me know.

**Reply by dam-ease** [reply]
I see your recent build has been successful.
I'll be closing this thread for now!
Happy building
